Lib/datetime.py

```diff
@@ -454,20 +454,23 @@ def __new__(cls, days=0, seconds=0, microseconds=0,
         return self
 
     def __repr__(self):
-        if self._microseconds:
-            return "%s.%s(%d, %d, %d)" % (self.__class__.__module__,
-                                          self.__class__.__qualname__,
-                                          self._days,
-                                          self._seconds,
-                                          self._microseconds)
+        args = []
+
```

Comment:
IMHO the empty lines doesn't help readability, you can remove them in all the code filling args.